src/index.ts

```diff
@@ -55,12 +55,15 @@ type Res = {
 
 export type Cast = typeof cast
 type Format = typeof format
+type Variables = Record<string, any>
```

Comment:
This should be explicitly `Record<string, string>` since that's what the protobuf type for this is going to be.

Reply:
I think also as we agreed,we probably want to lock down the key here to a finite number of keys that we want to support first, until we get more feedback.

We're defintiely going to do this on the server side, so not sure if we'd want to explicitly use some union type or whatever for the keyinstead of a freeform string.

Reply:
Yeah, at least to the API. Setting boost is: ```variables: {"@@boost_cached_queries": "true"}``` I'm not sure how these actually are converted, but maybe it's the same logic that we can use from `sanitize` internally? I haven't really played around at what other types we can use here besides some basics. It might just need to similarly go through sanitize like normal SQL params.

Reply:
So I guess up to you if you want the Variables itself to accept `any` and run it always through `sanitize` pass or not, but the underlying RPC call will require a string like a SQL string.